spec/models/ability_spec.rb

```diff
@@ -2,36 +2,23 @@
 require 'cancan/matchers'
 
 describe Ability do
-  describe 'for regular user' do
-    let(:user) { FactoryGirl.create(:user) }
-
-      it 'allows to crud comments' do
-        ability = Ability.new(user)
-        comment = Comment.new(user: user)
-        expect(ability).to be_able_to(:manage, comment)
-      end
+  let(:subject) { Ability.new(user) }
+  let(:comment) { Comment.new(user: user) }
 
-      it 'can not destroy user' do
-        ability = Ability.new(user)
-        comment = Comment.new(user: user)
-        expect(ability).to_not be_able_to(:destroy, user)
-      end
+  describe 'for regular user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.